stories/Appointment.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import { Story, Meta } from '@storybook/react'
+
+import { Appointment as AppointmentComponent, AppointmentProps } from '../components/Appointment'
+
+export default {
+  title: 'Component/Atoms/Appointment',
+  component: AppointmentComponent,
+  argTypes: {
+    date: {
+      description: 'Please ignore the time picker',
+      control: {
+        type: 'date',
+      },
+    },
+    start: {
+      name: 'start time',
+      table: {
+        type: {
+          summary: 'number',
+        },
+      },
+      control: {
+        type: 'number',
+        min: 1,
+        max: 12,
+      },
+    },
+    end: {
+      name: 'end time',
+      table: {
+        type: {
+          summary: 'number',
+        },
+      },
+      control: {
+        type: 'number',
+        min: 1,
+        max: 12,
```

Comment:
these don't seem to stop me:

> Friday, July 30, 2021, between 12–15 p.m. Pacific time

Reply:
Hmmmm. I'm going to go ahead and merge this without resolving it. I'll open a follow up (low priority) ticket.

Reply:
Ticket is #350